src/agent/rustjail/src/container.rs

```diff
@@ -682,7 +835,18 @@ fn do_init_child(cwfd: RawFd) -> Result<()> {
 
     // notify parent that the child's ready to start
     write_sync(cwfd, SYNC_SUCCESS, "")?;
-    log_child!(cfd_log, "ready to run exec");
+
+    Ok((args, oci_process, (*linux).clone()))
+}
+
+pub fn do_child_setup_release(
```

Comment:
What does the suffix `_release` mean?

Reply:
do_setup_wasm need return status and something back through the fifo.
So I moved fifo release code and something that have related with them from do_child_setup to do_child_setup_release.